lib/debug/server.rb

```diff
@@ -4,6 +4,8 @@
 require 'etc'
 require_relative 'config'
 require_relative 'version'
+require 'json'
```

Comment:
Please require them in line 165 because they are not used on other type connection.